tests/python/pants_test/engine/test_rules.py

```diff
@@ -839,7 +839,7 @@ def h() -> A:
       @rule
       def g() -> A:
         # This is a yield statement without an assignment, and not at the end.
```

Comment:
```suggestion
        # This is an await statement without an assignment, and not at the end.
```

Reply:
There may be a couple more of these. I recommend rip grepping `yield` - you'll have a lot of noise but could find some remaining instances.

Reply:
I did, I use https://github.com/cosmicexplorer/helm-rg and it is the root of all my power, just didn't consider documentation and expected changing the rule types to automatically fix things!